ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-reviewer.yml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-ci-reviewer
+spec:
+  params:
+    - name: pipeline_image
+    - name: git_username
+    - name: operator_name
+    - name: pr_url
+    - name: github_token_secret_key
+      description: The key within the Kubernetes Secret that contains the GitHub token.
+      default: token
+    - name: github_token_secret_name
+      description: The name of the Kubernetes Secret that contains the GitHub token.
+      default: github
+  workspaces:
+    - name: repo
+      description: Cloned repository
+  results:
+    - name: author_is_reviewer
+      description: >-
+        Check if the author of the PR is listed also as a reviewer
+    - name: list_reviewers
+      description: >-
+        Return list of all reviewers from ci.yaml file except author of the pull request
+  steps:
+    - name: get-ci-reviewer
+      image: "$(params.pipeline_image)"
+      workingDir: $(workspaces.repo.path)
+      script: |
+        #! /usr/bin/env bash
+        set -e
+        CI="operators/$(params.operator_name)/ci.yaml"
+        if [ ! -e "$CI" ]; then
+          echo >$(results.list_reviewers.path)
+          echo false >$(results.author_is_reviewer.path)
+        else
+          if grep -q 'reviewers:' $CI; then
+            yq -r '.reviewers[]' $CI >$(results.list_reviewers.path)
+            yq -r ".reviewers|index(\"$(params.git_username)\") != null" $CI >$(results.author_is_reviewer.path)
+          else
+            echo >$(results.list_reviewers.path)
+            echo false >$(results.author_is_reviewer.path)
+          fi
+        fi
+    - name: approve-pull-request
+      env:
+        - name: GITHUB_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: $(params.github_token_secret_name)
+              key: $(params.github_token_secret_key)
+      image: "$(params.pipeline_image)"
+      script: |
+        #! /usr/bin/env bash
+        set -e
+        # do not use -x to not leak github secrets into logs!
```

Comment:
We are not handling the github token directly anymore, you can re-enable the `-x` if you want.